### PR TITLE
Add usage example to tf.image.draw_bounding_boxes

### PR DESCRIPTION
**Pull request by eospi**
Add an example to `tf.image.draw_bounding_boxes` showing how to draw a bounding box in the center of an image.

### CODE REVIEW

**Review by alextp — CHANGES_REQUESTED** [review]
Can you start with a zero image instead of a random one, and can you make it small enough (say 3x3) that we can print the result so this becomes a doctest?

**Comment by eospi** [comment]
Thanks for looking over the PR! I've made the changes.

**Review by alextp — DISMISSED** [review]

**Review by mihaimaruseac — APPROVED** [review]

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -4044,6 +4044,28 @@ def draw_bounding_boxes_v2(images, boxes, colors, name=None):
 
   Returns:
     A `Tensor`. Has the same type as `images`.
+
+  Usage Example:
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: Add an empty line after `Usage Example:` to make the doctest show properly on the documentation website.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/image_ops_impl.py (hunk 1 above)
Comment (eospi) [reply tensorflow/python/ops/image_ops_impl.py]: I noticed the test fails after the first empty line in the doctest output. Does adding `<BLANKLINE>` in the empty lines fix that? 
```python
  <tf.Tensor: shape=(1, 3, 3, 3), dtype=float32, numpy=
  array([[[[1., 0., 0.],
          [1., 0., 0.],
          [1., 0., 0.]],

          [[1., 0., 0.],
          [0., 0., 0.],
          [1., 0., 0.]],

          [[1., 0., 0.],
          [1., 0., 0.],
          [1., 0., 0.]]]], dtype=float32)>
```

Comment (mihaimaruseac) [reply tensorflow/python/ops/image_ops_impl.py]: I think you don't need the empty lines in the output. At least I know that removing the empty lines fixed other PRs

Comment (eospi) [reply tensorflow/python/ops/image_ops_impl.py]: OK, I will remove those empty lines. Thanks!

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -4058,11 +4059,11 @@ def draw_bounding_boxes_v2(images, boxes, colors, name=None):
   array([[[[1., 0., 0.],
           [1., 0., 0.],
           [1., 0., 0.]],
-
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: You'll have to remove them here too. And in the other example below

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**